.circleci/config.yml

```diff
@@ -169,6 +169,10 @@ jobs:
           name: Trigger the version upgrade and release build
           command: |
             cd ..
+            ls -lah
+            SANITIZED_BRANCH=$(echo "$CIRCLE_BRANCH" | iconv -t ascii//TRANSLIT | sed -r s/[^a-zA-Z0-9\.]+/-/g | sed -r s/^-+\|-+$//g | tr A-Z a-z)
+            docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:${SANITIZED_BRANCH}-sha-${CIRCLE_SHA1}
+            docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$SANITIZED_BRANCH
```

Comment:
I am not sure how this will work because we are not really pushing the image to remote..

Reply:
This should pre-tag the image so that it is available for use with the `on-prem` binary.